geomesa-spark/geomesa-spark-sql/src/main/scala/org/apache/spark/sql/SQLGeometricOutputFunctions.scala

```diff
@@ -0,0 +1,50 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.apache.spark.sql
+
+import java.io.StringReader
+
+import com.vividsolutions.jts.geom.{Geometry, Point}
+import org.geotools.geojson.geom.GeometryJSON
+import org.locationtech.geomesa.utils.geohash.GeoHash
+import org.locationtech.geomesa.utils.text.WKBUtils
+
+
+object SQLGeometricOutputFunctions {
+  val geomJSON = new GeometryJSON()
```

Comment:
Is GeometryJSON thread safe?  If so, add a comment reminding us that it is.  Otherwise, I might recommend using a ThreadLocal here.